### DISCUSSION TITLE
Assigning Expression Pedals to Specific Keyboards

### DISCUSSION THREAD

**Discussion by DELeenhouts** [GrandOrgue/grandorgue#1029] [Help]
Hello,
I am new to GO and I am in the process of adding MIDI to my father's old Schober organ console.  I have it working . . . but not exactly as I would like.  The organ has 2 expression pedals, one for the Great manual and the other for the Swell manual.  
I am using the Grand Orgue sample organ which conveniently has 2 expression pedals . . . the left marked "R" and the right marked "GO".
My question is this:  How do I assign a specific expression pedal to a specific manual?
Thank you,
DELeenhouts

**Comment by oleg68** [comment]
1. You need the version 3.6.2-1 of GrandOrgue where the bug #1000 has been fixed
2. Right-click on each expression pedal picture, click on ``Listen for event`` and move your proper expression pedal. Then click to `OK`.
3. Check that it is working. If so, do ``File->Save``

**Comment by DELeenhouts** [comment]
Thank You Oleg,
I have noticed what appears to be a software "bug" with the demo Grand
Orgue v3.6.2-1 . . . I would like to notify someone who could determine if
indeed there is a bug and then address it.
Would that person be you?
If not, whom should I notify?
Thank you again for you assistance,
Dave Leenhouts

**Reply by oleg68** [reply]
@DELeenhouts 

The bug existed before this version. And it was fixed in 3.6.2-1, so this version shoud operate correctly with midi events.

If you found the bug still exists, please, explain here, what goes wrong.

**Comment by DELeenhouts** [comment]
Oleg,
I believe 3.6.2-1 works correctly regarding bug fix #1000 . . . I am now
referring to what appears to be a different bug in which not all of the
stops respond to the expression pedals.
For example, Montre 8' and Contrebasse 16' do not seem to respond to either
expression pedal.
Thank you,
Dave

**Comment by oleg68** [comment]
I can confirm that `Montre 8'` in the demo organ doesn' react to the `GO` swell pedal position, when `Flute Harm. 8'` works as expected.

@larspalo is it a desired behavior or a bug of the Demo Organ?

**Comment by DELeenhouts** [comment]
Oleg / Lars,
Thank you for asking whether the behavior I observed is desired or a bug.
I am a retired engineer and happen to notice inconsistencies but I do not
know what behavior is intentional and what is a software "bug".  Although
in this case it does seem that the volume of every stop should be
controllable with the expression pedals.
Thanks again,
Dave Leenhouts

**Reply by ahall41** [reply]
If you go to the Organ Settings tab, you will see that Montre 8 is in the Grand Orgue Unenclosed Windchest, therefore (by design) is not affected by the GO swell.

**Comment by DELeenhouts** [comment]
Thank you very much!
Dave Leenhouts
